From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: David Greaves <david@dgreaves.com>
Date: Fri, 12 Apr 2019 20:28:32 +0100
Subject: [PATCH] Skip semaphore test.

OBS's arm build environment doesn't have /dev/shm
mounted (real devices do) so this fails, crippling
multiprocessing support.

Author: Mike Sheldon <elleo@gnu.org>  2013-12-27 04:37:11
Committer: Thomas Perl <m@thp.io>  2013-12-27 08:06:55

Signed-off-by: David Greaves <david.greaves@jolla.com>
---
 configure.ac | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index bbe7f891e7..f177af2026 100644
--- a/configure.ac
+++ b/configure.ac
@@ -5543,7 +5543,7 @@ AC_CACHE_CHECK([whether POSIX semaphores are enabled], [ac_cv_posix_semaphores_e
     ])
   ],
   [ac_cv_posix_semaphores_enabled=yes],
-  [ac_cv_posix_semaphores_enabled=no],
+  [ac_cv_posix_semaphores_enabled=yes],
   [ac_cv_posix_semaphores_enabled=yes])
 )
 AS_VAR_IF([ac_cv_posix_semaphores_enabled], [no], [
@@ -5580,8 +5580,8 @@ AC_CACHE_CHECK([for broken sem_getvalue], [ac_cv_broken_sem_getvalue],
     ])
   ],
   [ac_cv_broken_sem_getvalue=no],
-  [ac_cv_broken_sem_getvalue=yes],
-  [ac_cv_broken_sem_getvalue=yes])
+  [ac_cv_broken_sem_getvalue=no],
+  [ac_cv_broken_sem_getvalue=no])
 )
 AS_VAR_IF([ac_cv_broken_sem_getvalue], [yes], [
   AC_DEFINE(
